llarp/router/router.hpp

```diff
@@ -295,10 +311,7 @@ namespace llarp
     void
     GossipRCIfNeeded(const RouterContact rc) override;
 
-    Router(
-        std::shared_ptr<llarp::thread::ThreadPool> worker,
-        llarp_ev_loop_ptr __netloop,
-        std::shared_ptr<Logic> logic);
+    explicit Router(llarp_ev_loop_ptr __netloop, std::shared_ptr<Logic> logic);
```

Comment:
double underscore?

Reply:
oh, that was from previous code... but yeah

Reply:
You should go epee style and use 5 underscores.